src/app/feed/feed.component.scss

```diff
@@ -53,3 +53,13 @@ app-footer.peeking{
 		}
 
 }
+
+.searching {
+	border-radius: 50%;
+	margin: 10% auto;
+	width: 100px;
+	height: 100px;
+	border: 5px solid rgb(216,27,96);
+	border-top-color: rgba(216,97,96,0.2);
+	animation: spin 1s infinite linear;
+}
```

Comment:
Also copy the spin animation from the feed-pagination scss file. That is why the animation is not working.